.github/workflows/rich-code-nav.yml

```diff
@@ -0,0 +1,44 @@
+name: Rich Code Navigation
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+      - stable
+
+env:
+  dotnet_sdk_version: '5.0.100'
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: NuGet Cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.nuget/packages
+          key: ${{ runner.os }}-nuget-${{ hashFiles('**/Directory.Build.targets') }}
+          restore-keys: |
+            ${{ runner.os }}-nuget-
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.dotnet_sdk_version }}
+
+      - name: Build
+        run: dotnet build Npgsql.sln --configuration Debug
+        shell: bash
+
+      - name: Rich Navigation Indexing
+        uses: microsoft/RichCodeNavIndexer@v0.1
+          with:
+            languages: csharp
+            repo-token: ${{ github.token }}
+          continue-on-error: true
```

Comment:
Isn't needed?

Reply:
Not following you.

Reply:
Why do you want to continue-on-error?

Reply:
That part came from examples I found since there's no docs how to set up that beast at least for non-MS devs. It was used in MS repos too which I mostly used to research how make it work.